java/examples/README.md

```
@@ -3,25 +3,32 @@
 ## Prerequisites
+
 - [CuVS libraries](https://docs.rapids.ai/api/cuvs/stable/build/#build-from-source)
 - Build the CuVS-Java API
 
 ## Run Examples
 
 ### CAGRA Example
+
 In the current directory do:
+
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)